helpers.sh

```diff
@@ -54,7 +54,13 @@ function repo_sync {
 		status "Adding git config extraheader for $domain"
 		git config --global http.https://${domain}.extraheader "$(cat /secrets/git.http.extraheader)"
 	fi
-	run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES}
+	for i in $(seq 4); do
+		run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
+		status "repo init failed with error $?"
+		[ $i -eq 4 ] && exit 1
```

Comment:
why don't return an actual code $?
After refactoring, we have it unchanged here since `run repo init`

Reply:
> why don't return an actual code $?

I can change it to exit with the actual code `$?` but there some other exits around on this script so it will not be useful for interpret by the caller (I think is jobserver)

> After refactoring, we have it unchanged here from run repo init

I didn't understand this part very well, can you say it another way please

Reply:
> I can change it to exit with the actual code `$?` but there some other exits around on this script so it will not be useful for interpret by the caller (I think is observer)

It is a usual approach to return an actual error code of a running command.  The caller may handle return codes in different ways, if not now, so in the future. Also, it generalizes code in this function (see line 73).

> I didn't understand this part very well, can you say it another way please

I mean, as now `run sleep` is not right after `run repo init`, we have an actual `$?` from `run repo init` when calling `exit`.

Reply:
I think with the status message and the error code we're good now, since in the end the job will be aborted.